csharp/src/Google.Protobuf.Benchmarks/ParseRawPrimitivesBenchmark.cs

```diff
@@ -46,15 +46,15 @@ namespace Google.Protobuf.Benchmarks
     public class ParseRawPrimitivesBenchmark
     {
         // key is the encodedSize of varint values
-        Dictionary<int, byte[]> varintInputBuffers;
+        Dictionary<int, byte[]> varintInputBuffers = new();
```

Comment:
Possibly add a comment to say that all of these initial values will be overwritten in GlobalSetup? (And make everything explicitly private.)